spec/mailers/tt_application_mailer_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe TTApplicationMailer, type: :mailer do
+  subject(:mailer) { described_class }
+
+  describe 'send_application' do
+    let(:applicant) do
+      {
+        first_name: 'Joe',
+        last_name: 'Bloggs',
+        preferred_name: 'Joe',
+        title: 'Prof',
+        date_of_birth: Date.new(1994, 2, 10),
+        email: 'joe.bloggs@example.com',
+        phone: '012345',
+        address: '123 Fake Street'
+      }
```

Comment:
It's great that we can test all these things flowing into the template, but as we don't know what's actually required yet, it feels a bit premature. Given that each additional test basically just asserts we've given ERB the right thing, could we radically cut this file down to just take a single piece of data? See also comment on using `applicant:` as the param name.